widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -5161,6 +5161,10 @@ button.adminButton {
   background-color: #fff;
 }
 
+:focus-visible .activedescendant {
```

Comment:
As suggested elsewhere, `active-descedant`.
Also, please, move this rule to after the section `/* FOCUS CONTAINER */`.

Reply:
addressed